### PR TITLE
Merge branch 'master' of https://github.com/Rahgooy/saul

### PR DESCRIPTION
**Pull request by umarmanzoor**
Merge branch 'master' of https://github.com/Rahgooy/saul

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala [hunk 1]
```diff
@@ -69,25 +69,42 @@ object MultiModalSpRLSensors {
...
+    "boat-rafting" -> "vehicle",
+    "polar-bear" -> "bear",
+    "fowl-hen" -> "bird",
+    "flock-of-birds" -> "animal",
+    "school-of-fishes" -> "animal",
+    "man-made-other" -> "man-made",
+    "ocean-animal" -> "animal",
+    "herd-of-mammals" -> "animal",
+    "mammal-other" -> "mammal"
   )
```
Comment (Rahgooy) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala:109]: @kordjamshidi are you ok with this?

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala:109]: the PR is ok to merge, but @Rahgooy  please check the `phraseConceptToWord = HashMap(...`

Comment (umarmanzoor) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala:109]: @kordjamshidi this hashmap has been created using clef image ontology.

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala:109]: that is ok. I just wanted to make sure the right side of the map does not come from the text.

**Pull request merged**